CONTRIBUTING.md

```diff
@@ -8,6 +8,13 @@ This guide details how to use issues and pull requests to improve KubeRay projec
 
 ## General Guidelines
 
+### Getting involved
```

Comment:
Maybe we can also add this info to [README](https://github.com/ray-project/kuberay/blob/master/README.md) to get broader visibility?